exercises/raindrops/description.md

```diff
@@ -1,16 +1,14 @@
-Convert a number to a string, the contents of which depend on the number's factors.
+In this variant of the children's game [Fizz Buzz](https://en.wikipedia.org/wiki/Fizz_buzz) -- which has often been used as a screening exercise in technical job interviews -- your task is to convert an number into a string that contains raindrop sounds corresponding to certain potential factors. A factor is a number that evenly divides into another number, leaving no remainder. The simplest way to test if a one number is a factor of another is to use the [modulo operation](https://en.wikipedia.org/wiki/Modulo_operation).
```

Comment:
> The simplest way to test if a one number is a factor of another is to use the [modulo operation](https://en.wikipedia.org/wiki/Modulo_operation).

That pretty much gives away the solution.

Reply:
It does give away how to determine if ie 3 is a factor of number, but it doesn't give away how to respond to that with "pling" ... in a sense forcing the student to discover _how_ to do modulo in their language is, I think, more important than figuring out _that_ they should use modulo. Given the position this is likely to take in a track it seems best to just tell them what operator / function to look for.